admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -43,6 +43,7 @@ export default {
   price: {
     priceTaxExcludedInput: '#product_price_price_tax_excluded',
   },
+  combinationsFiltersContainer: '#combinations_filters',
```

Comment:
this should go under "combinations" key (see line 86)

Reply:
indeed!